11-information_governance.Rmd

```diff
@@ -28,187 +28,119 @@ If requesting access to a data source, you should provide the GitHub usernames o
 * Files stored in users' home directories are backed up to Amazon S3 automatically.
 * Previous versions of files stored in users' home directories are also backed up to Amazon S3.
 * Once files are deleted from a user's home directory, the backup is retained for a further 90 days and can be restored.
-* To request that a file be restored or to access a previous version of a file, contact the [Analytical Platform team](mailto:analytical_platform@digital.justice.gov.uk).
+* To request that a file be restored or to access a previous version of a file, contact the Analytical Platform team at [analytical_platform@digital.justice.gov.uk](mailto:analytical_platform@digital.justice.gov.uk).
 
-#### Best practice guidelines
+## Data movements
 
-All users should comply with the following best practice guidelines for retaining personal data.
+You can move data classified as OFFICIAL and OFFICIAL-SENSITIVE onto the Analytical Platform. SECRET and TOP SECRET data is not allowed on the Analytical Platform.
```

Comment:
I think it's better to put 'data classified as OFFICIAL (including OFFICIAL-SENSITIVE)' since technically there are only three levels of data security (OFFICIAL, SECRET, TOP SECRET).

Reply:
Changed in fe48793.